backend/src/main/scala/cromwell/backend/BackendActor.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.backend
+
+import akka.actor.{ActorRef, Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendActor.{Validate, AfterAll, BeforeAll, CreateJobExecutor}
+import cromwell.backend.model.JobDescriptor
+
+object BackendActor {
```

Comment:
WorkflowExecutionActor => it may not execute the entire workflow.
WorkflowPreparationActor => this is not a workflow.
WorkflowLevelBackendSpecificPreparationCleanupAndJobDescriptorMakerActor => too long :)

I think Backend is representative.